ci/repipe

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -e
+
+if [[ $(which ytt) == "" ]]; then
+  echo "You will need to install ytt to repipe. https://carvel.dev/ytt/"
+  exit 1
+fi
+
+target="${FLY_TARGET:-galoy}"
+team=dev
+
+TMPDIR=""¬
```

Comment:
What does `¬` do here? Is that a key on your keyboard or did you have to get it from elsewhere? I don't have it on mine

Reply:
Good spot... its a copy paste mistake will get rid of it